kinder/cmd/kinder/create/cluster/createcluster.go

```diff
@@ -210,8 +210,8 @@ func getInitVersionFromImage(image string) (string, error) {
 			}
 		}
 
-		log.Debug("Failed to read initVersion from image name, using release/stable release")
-		return kextract.ResolveLabel("release/stable")
+		log.Debug("Failed to read initVersion from image name, using release/latest-1.14 release")
+		return kextract.ResolveLabel("release/latest-1.14")
```

Comment:
this should continue to be `release/stable`.
it's a standard fallback for cluster creation in kinder.

Reply:
done